script.js

```diff
@@ -2,6 +2,224 @@ const unionButton = document.querySelector('.union-button');
+const popup = document.querySelector('.popup');
+const over = document.querySelector('.over');
+seeproject.forEach((p) => p.addEventListener('click', (p) => {
+  const { id } = p.target;
+  const pop = projects.find((p) => p.id === id);
+  popup.innerHTML = `
+  <div class="titles">
+    <div class="flex-row" style="justify-content:space-between;">
+      <h2 class="tonic-1 ignore-mbl" style="padding-left:6%">${pop.title}</h2>
+      <h2 class="tonic-1 tonic-12  ignore-desk">${pop.title1}</h2>
+      <button class="close close12" style="border-color:white;">&times;</button>
+    </div>
+    <div class="Card_text flex-column">
+    <ul class="page page12 flex-row" style="padding-left:7%">
+      <li class="title_1 ignore-mbl">${pop.company}</li>
+      <li class="title_1  ignore-desk">${pop.company1}</li>
+      <li><img src="${pop.icon}" alt="Dot icon" ></li>
+      <li class="ignore-mbl">${pop.specialization}</li>
+      <li class="ignore-desk">${pop.specialization1}</li>
+      <li><img src="${pop.icon}" alt="Dot icon" ></li>
+      <li>${pop.year}</li>
+    </ul>
+    </div>
+    <img src="${pop.imageMobile}" alt="live" class="work" style="padding-left:6%; margin-top:0%">
+    <img src="${pop.imageDesktop}" alt="live" class="pic"  style="width:1256px; height:440px; margin: 20px;">
+    <div class="explain">
+      <p class="tonic-text2">
+        ${pop.description1}
+        <br class="ignore-desk">
+        <br class="ignore-desk">
+        <span class="ignore-desk">${pop.description1}</span>
+      </p>
+      <div class="smart">
+          <ul class="other-tag flex-row" style="padding-left:6%">
+            <li class="box">${pop.technologies.tech1}</li>
+            <li class="box_desk ignore-mbl">${pop.technologies.techE}</li>
+            <li class="box">${pop.technologies.tech2}</li>
+            <li class="box">${pop.technologies.tech3}</li>
+          </ul>
+          <ul class="other-tag flex-row" style="padding-left:6%">
+            <li class="box ignore-desk">Ruby</li>
+            <li class="box ignore-desk">Bootstrap</li>
+          </ul>
+          <div class="btns">
+            <button type="button" id="live" class="but">See live &nbsp;
+              <img src="./Properties/source.svg" alt="live" class = "live">
+            </button>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   
+            <button type="button" id="source" class="but">See source &nbsp;
+              <img src="./Properties/github.png" alt="source">
+            </button>
+          </div>
```

Comment:
Good job implementing the modal as demanded 👍. In order to present a good UI to visitors, I suggest that you verify the path for this image to ensure that it is correctly linked to. because it is a broken image on the UI.

<img width="284" alt="bad button" src="https://github.com/zuhebahmed88091/Portfolio_site/assets/84930355/f4dfeff9-3cc1-4cd7-bf0c-29b50ff0cbbc">

Reply:
from my browser icon is showing. how can i resolve this problem?

Reply:
and you can check i have added github.png file